guides/InstallationAdminGuide.md

```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
 Install the program, taking note of the name and location of the folder in which the avconv and avprobe executable files have been installed, as the openEQUELLA installation will require these details to properly configure and run openEQUELLA.
 
 ### To install Libav for Windows
-1. Go to <http://builds.libav.org/windows/release-gpl/> and download the relevant release build. (e.g. libav-11.3-win64.7z)
+1. Go to <http://builds.libav.org/windows/release-gpl/> and download the relevant release build. (e.g. libav-11.12-win64.7z)
```

Comment:
While good to use the latest point release, I don't know if folks have specifically tested 11.12.  Unless you know it works, I would not reference it here.

Reply:
I don't know if anyone has tested in production or under a heavy workload. I have only done a basic install for my test VM and haven't loaded any data yet.